### PR TITLE
UIDEXP-22: Implement logic for periodical fetching of running jobs and logs

### PR DESCRIPTION
**Pull request by VictorSoroka1**
## Purpose

Implement logic for periodical fetching of running jobs and logs in the scope of [UIDEXP-22](https://issues.folio.org/browse/UIDEXP-22).

### CODE REVIEW

File: src/components/Jobs/RunningJobs/RunningJobs.js [hunk 1]
```diff
@@ -1,28 +1,34 @@
...
   sortRunningJobs,
   JobsListAccordion,
 } from '@folio/stripes-data-transfer-components';
 
-import { JOB_STATUSES } from '../../../utils/constants';
+import { JOB_EXECUTION_STATUSES } from '../../../utils/constants';
 import ItemFormatter from './ItemFormatter';
-import tempData from './tempData';
+import { DataFetcherContext } from '../../../contexts/DataFetcherContext';
+import { sampleJobExecution } from '../../../../test/bigtest/network/scenarios/fetch-job-profiles-success';
```
Comment (YevheniiMaltsev) [inline src/components/Jobs/RunningJobs/RunningJobs.js]: As far as understand the data from tempData was moved fetch-job-profiles-success. Could you please clarify if tempData was deleted?

File: test/bigtest/network/scenarios/fetch-job-profiles-success.js [hunk 2]
```diff
@@ -1,28 +1,28 @@
...
   },
   progress: {
-    jobExecutionId: 'de96f7a1-8706-42e9-ba86-c44c44592511',
     current: 21650,
     total: 45265,
   },
   startedDate: '2020-01-27T10:10:18.743+0000',
-  status: 'PARSING_IN_PROGRESS',
-  uiStatus: 'RUNNING',
+  completedDate: '2019-10-30T12:40:01.000',
```
Comment (YevheniiMaltsev) [inline test/bigtest/network/scenarios/fetch-job-profiles-success.js]: Could we have completedDate field if the job is in progress (According to the status in line 19)?

File: src/routes/Home.js [hunk 3]
```diff
@@ -5,39 +5,80 @@ import {
...
+        </Pane>
+      </DataFetcher>
     </Paneset>
   );
 }
+
+Home.manifest = Object.freeze({
+  jobs: {
+    type: 'okapi',
+    path: jobsUrl,
```
Comment (YevheniiMaltsev) [inline src/routes/Home.js]: It seems, we can move jobsURL and logsURL closer to their usage. What do you think about it?

File: test/bigtest/interactors/jobs/RunningJobsInteractor.js [hunk 4]
```diff
@@ -22,3 +22,11 @@ export class RunningJobInteractor {
     return this.when(() => this.isPresent);
   }
 }
+
+@interactor
+export class RunningJobsInteractor {
```
Comment (YevheniiMaltsev) [inline test/bigtest/interactors/jobs/RunningJobsInteractor.js]: What about RunningJobsContainer or something like this one? Or maybe find a better name for RunningJobInteractor? It's really challenging to find difference between RunningJobInteractor and RunningJobsInteractor.

File: test/bigtest/tests/jobs-test.js [hunk 5]
```diff
@@ -40,46 +40,50 @@ describe('Jobs lists', () => {
       expect(jobs.accordions(0).title.text).to.equal(translation.runningJobs);
     });
 
+    it('should display correct amount of running jobs', () => {
+      expect(runningJobs.jobItemsAmount).to.equal(3);
```
Comment (YevheniiMaltsev) [inline test/bigtest/tests/jobs-test.js]: Thanks for adding tests for multiple running jobs!

File: src/routes/Home.js (hunk 3 above)
Comment (VictorSoroka1) [reply src/routes/Home.js]: Sounds good.

File: test/bigtest/tests/jobs-test.js (hunk 5 above)
Comment (VictorSoroka1) [reply test/bigtest/tests/jobs-test.js]: Thank you.

File: test/bigtest/network/scenarios/fetch-job-profiles-success.js (hunk 2 above)
Comment (VictorSoroka1) [reply test/bigtest/network/scenarios/fetch-job-profiles-success.js]: Good point. In this particular case, it makes sense to remove it.

File: src/components/Jobs/RunningJobs/RunningJobs.js (hunk 1 above)
Comment (VictorSoroka1) [reply src/components/Jobs/RunningJobs/RunningJobs.js]: I have firstly moved the file to the scenarios folder and then removed. Seems like github has identified that file was removed, but the `tempData` indeed is removed now.

File: test/bigtest/interactors/jobs/RunningJobsInteractor.js (hunk 4 above)
Comment (VictorSoroka1) [reply test/bigtest/interactors/jobs/RunningJobsInteractor.js]: Yeah, it can be challenging indeed. Will use `RunningJobsContainer` instead.

Comment (YevheniiMaltsev) [reply test/bigtest/interactors/jobs/RunningJobsInteractor.js]: Sounds good. Thanks

**Pull request merged**